packages/terra-alert/src/terra-dev-site/doc/example/ActionAndDismissibleExample.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import Button from 'terra-button';
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
+import Alert from 'terra-alert/lib/Alert';
+
+class ActionAndDismissibleExample extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isDismissed: false,
+      actionButtonClickCount: 0,
+    };
+    this.handleDismiss = this.handleDismiss.bind(this);
+    this.actionFunc = this.actionFunc.bind(this);
+  }
+
+  handleDismiss() {
+    const newState = this.state;
+    newState.isDismissed = true;
```

Comment:
This is actually changing the state I believe. If you are looking for this approach, you need to duplicate the object on line 18.

Reply:
Actually, why not just do this?

```jsx
this.setState({
   isDismissed: true
});
```

Reply:
This has been carried forward from the previous example. If we want to change it I think it would be best if we do that in a separate pull request.

Reply:
I noticed a lot of my comments were curtailed around old examples. I'm ok with leaving those off for future enhancements.